modules/setting-higher-pid-limit-on-machinepool.adoc

```diff
@@ -0,0 +1,84 @@
+// Module included in the following assemblies:
+//
+// * rosa_cluster_admin/rosa-configuring-pid-limits.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="setting-higher-pid-limit-on-machine-pool_{context}"]
+= Setting a higher process ID limit on a machine pool in a {product-title} cluster
+
+You can set a higher `podPidsLimit` for machine pools in an existing {product-title} (ROSA) cluster by creating or editing a `KubeletConfig` object that changes the `--pod-pids-limit` parameter.
+
+[IMPORTANT]
+====
+Changing the `podPidsLimit` on an existing machine pool will trigger nodes in the machine pool to reboot one at a time. Make this change outside of peak usage hours for workloads in your machine pool and avoid upgrading or hibernating your cluster until all nodes have rebooted.
+====
+
+.Prerequisites
+
+* You have a {product-title} cluster.
+* You have installed the ROSA CLI (`rosa`).
+* You have logged in to your Red Hat account by using the ROSA CLI.
+
+.Procedure
+
+. Create a new `KubeletConfig` object for your cluster that specifies a new `--pod-pids-limit`.
++
+[source,terminal]
+----
+$ rosa create kubeletconfig -c <cluster_name> --name=<kubeletconfig_name>  --pod-pids-limit=<value>
+----
++
+For example, the following command creates a `set-high-pids` `KubeletConfig` object for the `my-cluster` cluster that sets a maximum of 16,384 PIDs per pod:
++
+[source,terminal]
+----
+$ rosa create kubeletconfig -c my-cluster --name=set-high-pids --pod-pids-limit=16384
+----
+
+. Associate the new `KubeletConfig` object with a new or existing machine pool.
```

Comment:
Based on IBMSG "Procedure": https://www.ibm.com/docs/en/ibm-style?topic=punctuation-colons#introductory-text 

```suggestion
. Associate the new `KubeletConfig` object with a new or existing machine pool:
```

Reply:
Ignoring this one because the colon is attached to the conditional instructions, e.g. "For an existing machine pool:"